lib/web_ui/lib/src/engine/skwasm/skwasm_impl/image.dart

```diff
@@ -74,6 +74,9 @@ class SkwasmImage extends SkwasmObjectWrapper<RawImage> implements ui.Image {
   @override
   List<StackTrace>? debugGetOpenHandleStackTraces() => null;
 
+  @override
+  int get approximateBytesUsed => width * height * 4;
```

Comment:
@eyebrowsoffire - we should make a call into Skia for this value but I'm not sure how that works in the skwasm backend.

Reply:
We'll have to pipe a binding through `raw_image.dart` and `image.cpp` to get the value from skia in the skwasm renderer. This should look very similar to the `approximateBytesUsed` call in `picture.dart`/`raw_picture.dart`/`picture.cpp` we have already implemented in the skwasm renderer.